Ticket #3003 Autocomplete clipping long text

olgamelnichuk opened the pull request:
@alf239, @rpetry - please, review

Commit:
Ticket #3003 Autocomplete clipping long text

alf239 commented:
We have absolutely the same problem with genes (e.g. if you enter `i`, you'll get _I-2_ gene with _Drosophila melanogaster_ clipped on the right)

Commit:
Ticket #3003 Autocomplete clipping long text
added tooltip for gene autocomplete box

alf239 commented:
For `I-2`, `&nbsp;` gets escaped in the pop up. I suggest we stop and discuss what exactly we're trying to achieve — it seems we've got a broken telephone effect here.

Commit:
Ticket #3003 Autocomplete clipping long text
- showing only first matched alternative term in the tooltip
- showing EFO id and factor name in the tooltip

Commit:
Ticket #3003 Autocomplete clipping long text
&nbsp; removed
